srcpkgs/git-credential-oauth/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'git-credential-oauth'
+pkgname=git-credential-oauth
+version=0.8.0
+revision=1
+build_style=go
+build_helper=qemu
```

Comment:
Is qemu necessary? I copied from another Go template

Reply:
not here, it's used for the `vtargetrun` function to run, for example to run a built binary to generate shell completions

Reply:
Thanks